README.md

```diff
@@ -193,6 +193,30 @@ Add 21% tax and overhead cost stored in a helper
 
 ## Other
 
+### Extra state attributes
+
+- ```average```: An average of every today
+- ```off_peak_1```: Off peak 1 refers to the average price of the time period from midnight to 8am
+- ```off_peak_2```: Off peak 2 refers to the average price of the time period from 8pm to midnight
+- ```peak```: Peak refers to the average price of the period from 8am to 8pm
+- ```min```: Lowest value today
+- ```max```: Highest value today
+- ```mean```: Actually just median of all values
+- ```unit```: what that of Unit, eg: kWh
+- ```currency```: What type of Currency
+- ```country```: What Country data is fetched for
+- ```region```: The specific region of prices
+- ```low_price```: If current hour is the lowest of the day
```

Comment:
I believe this is not true. The value is true if the price is below the low_price threshold set in the config, as a factor of the average price.

Reply:
this is correct